src/tests/ie_test_utils/functional_test_utils/test_runner/run_conf.py

```diff
@@ -0,0 +1,162 @@
+from datetime import datetime, timedelta
+import subprocess
+import argparse
+import re
+
+class Runner():
+    def __init__(self, path_to_bin, args):
+        self.tests_list = []
+        self.tests_goups = []
+
+        self.path_to_bin = path_to_bin
+        self.args = args
+
+        self.failed = 0
+        self.skipped = 0
+        self.passed = 0
+        self.crashed = 0
+        self.hanged = 0
+        self.time = timedelta(seconds=0)
+
+    def collect_tests_list(self):
+        args_list = [self.path_to_bin]
+        args_list.extend(self.args.split(" "))
+        args_list.append("--gtest_list_tests")
+
+        process = subprocess.Popen(args_list,
+                                   stdout=subprocess.PIPE,
+                                   stderr=subprocess.PIPE)
+        out, err = process.communicate()
+        exit_code = process.returncode
+        if exit_code != 0:
+            print(f"ERR happend on getting gtest list tests: {err}")
+            exit(1)
+
+        prefix = ""
+        for line in out.decode('ascii').split('\n'):
+            if line:
+                if "# GetParam()" in line:
+                    self.tests_list.append(prefix + line.split("# GetParam()")[0].strip())
+                else:
+                    prefix = line
+
+    def collect_tests_goups(self):
+        groups = {}
+        i = 0
+        for test in self.tests_list:
+            test_type = test.split('.')[0]
+            groups[test_type] = i
+
+            # try to collect group by operation and PRC
+            matches = re.match(r"(.*PRC=.*_IR_name=(.*))_\d*.xml", test)
```

Comment:
Do not forget about `OpImplCheck` please qand other. The script can be reused for `APIConformance`